### PR TITLE
discard time zone when encoding time

### PR DESCRIPTION
**Pull request by tamird**
cc @vivekmenezes

### CODE REVIEW

File: proto/data.go [hunk 1]
```diff
@@ -372,12 +368,11 @@ func (v *Value) GetInt() (int64, error) {
...
 func (v *Value) GetTime() (time.Time, error) {
-	var t time.Time
-	if v.GetTag() != ValueType_TIME {
-		return t, fmt.Errorf("value type is not TIME: %s", v.GetTag())
+	if tag := v.GetTag(); tag != ValueType_TIME {
+		return time.Time{}, fmt.Errorf("value type is not TIME: %s", tag)
 	}
-	err := t.UnmarshalBinary(v.Bytes)
-	return t, err
+	_, t := encoding.DecodeTime(v.Bytes)
```
Comment (tschottdorf) [inline proto/data.go]: panic if the first thing (presumably an error) comes back unexpected?

Comment (tschottdorf) [inline proto/data.go]: ah, nevermind. It's a slice. Probably not worth the assertion then.

Comment (vivekmenezes) [inline proto/data.go]: is there a reason why we don't return an error when we hit a problem with decoding here and for the other types above?

**Comment by vivekmenezes** [comment]
LGTM

File: proto/data.go (hunk 1 above)
Comment (vivekmenezes) [inline proto/data.go]: The reason I ask is because the data is coming from a storage device and anything can go wrong there.

Comment (tamird) [inline proto/data.go]: The decoding routines all panic instead of returning errors, I believe.

Comment (tamird) [inline proto/data.go]: It's important to remember that `proto.Value` carries a checksum, but we don't seem to be using it consistently at all.

Comment (tamird) [inline proto/data.go]: Ah, I stand corrected, we are using the checksums correctly.

**Comment by petermattis** [comment]
LGTM

**Pull request merged**